src/main/webapp/transcript.js

```diff
@@ -58,27 +59,49 @@ function addMultipleTranscriptLinesToDom(transcriptLines) {
 }
 
 /**
- * Creates an <li> element containing {@code transcriptLine}'s text and
- * start time, and appends it to {@code ulElement}.
+ * Creates an <li> element containing {@code transcriptLine}'s text, start time,
+ * and end time and appends it to {@code ulElement}.
  */
 function appendTextToList(transcriptLine, ulElement) {
   const liElement = document.createElement('li');
-  const infoDivElement = document.createElement('div');
+  const startDate = new Date(transcriptLine.start);
+  const endDate = new Date(transcriptLine.end);
+  const startTimestamp = `${startDate.getHours()}:${startDate.getMinutes()}:${
+      startDate.getSeconds()}`;
+  const endTimestamp =
+      `${endDate.getHours()}:${endDate.getMinutes()}:${endDate.getSeconds()}`;
+  const timestamp = `${startTimestamp} - ${endTimestamp}`;
 
-  appendPTagToContainer(transcriptLine.start, infoDivElement);
-  liElement.appendChild(infoDivElement);
-  appendPTagToContainer(transcriptLine.content, liElement);
+  const timestampPElement = appendParagraphToContainer(timestamp, liElement);
```

Comment:
Why is this named PElement. I thought this was creating a li element.

Reply:
Resolved in chat.

`appendParagraphToContainer` was refactored to include an optional argument for tag classes.